src/apps/service/apps/agave.clj

```diff
@@ -139,61 +180,106 @@
        :documentation ""
        :references    []}))
 
+  (getAppDocs [_ system-id app-id]
+    (validate-system-id system-id)
+    {:app_id        app-id
+     :documentation ""
+     :references    []})
+
   (getAppIntegrationData [_ app-id]
     (when-not (util/uuid? app-id)
       (service/bad-request integration-data-rejection)))
 
+  (getAppIntegrationData [_ system-id app-id]
+    (validate-system-id system-id)
+    (service/bad-request integration-data-rejection))
+
   (getToolIntegrationData [_ tool-id]
     (when-not (util/uuid? tool-id)
       (service/bad-request integration-data-rejection)))
 
+  (getToolIntegrationData [_ system-id tool-id]
+    (validate-system-id system-id)
+    (service/bad-request integration-data-rejection))
+
   (updateAppIntegrationData [_ app-id integration-data-id]
     (when-not (util/uuid? app-id)
       (service/bad-request integration-data-rejection)))
 
+  (updateAppIntegrationData [_ system-id app-id integration-data-id]
+    (validate-system-id system-id)
+    (service/bad-request integration-data-rejection))
+
   (updateToolIntegrationData [_ tool-id integration-data-id]
     (when-not (util/uuid? tool-id)
       (service/bad-request integration-data-rejection)))
 
+  (updateToolIntegrationData [_ system-id tool-id integration-data-id]
+    (validate-system-id system-id)
+    (service/bad-request integration-data-rejection))
+
   (ownerEditAppDocs [_ app-id _]
     (when-not (util/uuid? app-id)
       (reject-app-documentation-edit-request)))
 
+  (ownerEditAppDocs [_ system-id app-id _]
+    (validate-system-id system-id)
+    (reject-app-documentation-edit-request))
+
   (ownerAddAppDocs [_ app-id _]
     (when-not (util/uuid? app-id)
       (reject-app-documentation-edit-request)))
 
+  (ownerAddAppDocs [_ system-id app-id _]
+    (validate-system-id system-id)
+    (reject-app-documentation-edit-request))
+
   (adminEditAppDocs [_ app-id _]
     (when-not (util/uuid? app-id)
       (reject-app-documentation-edit-request)))
 
+  (adminEditAppDocs [_ system-id app-id _]
+    (validate-system-id system-id)
+    (reject-app-documentation-edit-request))
+
   (adminAddAppDocs [_ app-id _]
     (when-not (util/uuid? app-id)
       (reject-app-documentation-edit-request)))
 
+  (adminAddAppDocs [_ system-id app-id _]
+    (validate-system-id system-id)
+    (reject-app-documentation-edit-request))
```

Comment:
Are there service tests for these endpoints which return `bad requests`? Seems like easy pickings.